packages/react-core/src/components/ClipboardCopy/__tests__/ClipboardCopy.test.tsx

```diff
@@ -336,6 +336,21 @@ test('Does not call onCopy when ClipboardCopyButton is not clicked', async () =>
   expect(onCopyMock).not.toHaveBeenCalled();
 });
 
+test('Can take array of strings as children', async () => {
+  const onCopyMock = jest.fn();
+  const user = userEvent.setup();
+  const childrenArray = children.split(' ');
+
+  // sanity check to ensure we are checking an array with at least 2 elements
+  expect(childrenArray.length).toBeGreaterThan(1);
+
+  render(<ClipboardCopy onCopy={onCopyMock}>{childrenArray}</ClipboardCopy>);
+
+  await user.click(screen.getByRole('button', { name: 'Test CCB clicker' }));
+
+  expect(onCopyMock).toHaveBeenCalledWith(children);
```

Comment:
```suggestion
  expect(onCopyMock).toHaveBeenCalledWith(expect.any(Object), children);
```

This would help the test pass. The details of the event object may not really be all that important for the purposes of this test, since we just want to check that an array of children are being passed to the `onCopy` handler as expected.